GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/CarStorePageController.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class CarStorePageController : MonoBehaviour
+{
+    public GameObject itemSedan;
+    public GameObject itemTruck;
+    public GameObject itemJeep;
+    public GameObject itemKart;
+    public GameObject confirmPanel;
+    public Text confirmText;
+
+    private int _priceInCoin;
+    private string _itemNameToPurchase;
+    private GameManager _gameManager;
+    
+    // Start is called before the first frame update
+    private void Start()
+    {
+        _gameManager = FindObjectOfType<GameManager>();
+        RefreshPage();
+    }
+
+    // refresh the page
+    private void RefreshPage()
+    {
+        confirmPanel.SetActive(false);
+        // check if player already owns the car
+        CheckCarOwnership("carSedan", itemSedan, true);
+        CheckCarOwnership("carTruck", itemTruck, false);
+        CheckCarOwnership("carJeep", itemJeep, true);
+        CheckCarOwnership("carKart", itemKart, true);
+    }
+    
+    
+    public void ItemSedanOnclick()
+    {
+        
+        itemCoinsOnClick("carSedan", 0f, false);
+        _itemNameToPurchase = "carSedan";
+    }
+    
+    public void ItemTruckOnClick()
+    {
+        if (PlayerPrefs.GetInt("coins", 20) >= 20)
```

Comment:
Don't hard code these "Jeep", "Truck", 20, 0, 2.99f... Instead, define some object as const in this class, or in some const data class.